Makefile

```diff
@@ -1,13 +1,44 @@
+# backwards compatibility name for CONTIV_INSTALLER_VERSION
+export BUILD_VERSION ?= devbuild
+# sets the version for the installer output artifacts
+export CONTIV_INSTALLER_VERSION ?= $(BUILD_VERSION)
+# downloaded and built assets intended to go in installer by build.sh
+export CONTIV_ARTIFACT_STAGING := $(PWD)/artifact_staging
+# some assets are retrieved from GitHub, this is the default version to fetch
+export DEFAULT_DOWNLOAD_CONTIV_VERSION := 1.1.5
+export NETPLUGIN_OWNER ?= contiv
+# setting NETPLUGIN_BRANCH compiles that commit on demand,
+# setting CONTIV_NETPLUGIN_VERSION will download that released version
+ifeq ($(NETPLUGIN_BRANCH),)
+export CONTIV_NETPLUGIN_VERSION ?= $(DEFAULT_DOWNLOAD_CONTIV_VERSION)
+else
+export CONTIV_NETPLUGIN_VERSION := $(NETPLUGIN_OWNER)-$(NETPLUGIN_BRANCH)
+endif
+export CONTIV_NETPLUGIN_TARBALL_NAME := netplugin-$(CONTIV_NETPLUGIN_VERSION).tar.bz2
+export CONTIV_ANSIBLE_COMMIT ?= 4e67f54a8042debfc3d8b504046d0a1d4ea38c37
```

Comment:
hard coded sha1?

Reply:
that actually is already merged to master, i will try rebasing